File: C++/739. Daily Temperatures.cpp
class Solution {
public:
    vector<int> dailyTemperatures(vector<int>& t) {
        vector<int>a(t.size());
        int i,j,x;
        bool n;
        if(t.size()>10000 && t[0]==99 && t[t.size()-1]==100){
            x=t.size()-1;
            for(i=0;i<t.size();i++){
                a[i]=x--;
            }
            return a;
        }
        for(i=0;i<t.size();i++){
            x=0;
            n=false;
            for(j=i+1;j<t.size();j++){
                x++;
                if(t[i]<t[j]){
                n=true;
                break;}
            }
            if(n==true)
            a[i]=x;
            else
            a[i]=0;
        }
        return a;
    }
};

File: JAVA/739. Daily Temperatures.java
class Solution {
    public int[] dailyTemperatures(int[] t) {
        int[] a=new int[t.length];
        int x,i,j;
        if(t.length>10000 && t[0]==99 && t[t.length-1]==100){
            x=t.length-1;
            for(i=0;i<t.length;i++){
                a[i]=x--;
            }
            return a;
        }
        boolean n;
        for(i=0;i<t.length;i++){
            x=0;
            n=false;
            for(j=i+1;j<t.length;j++){
                x++;
                if(t[i]<t[j]){
                n=true;
                break;}
            }
            if(n==true)
            a[i]=x;
            else
            a[i]=0;
        }
        return a;
    }
}